app/models.py

```diff
@@ -1498,6 +1498,15 @@ class Notification(db.Model):
             'notification_type',
             'status',
             'created_at'
+        ),
+        Index(
+            "ix_notifications_service_created_at_template_composite",
```

Comment:
Very optional: how about naming this index after the query? It's designed around that query and we don't expect it to be used for anything else. Richard was telling me that GOV.UK Pay have "an index for every query". We're not there yet, but this might be a helpful convention since we're adding more indices.

Possible name: `ix_notifications_template_statistics_for_service_by_day`. I think that's just short enough to be accepted.

Reply:
Excellent idea, love it. I struggled to name this due to the length constraint as I wanted to get every field in but that wasn't an option. This is a great alternative.